akka-http-caching/src/main/mima-filters/10.1.8.backwards.excludes

```diff
@@ -0,0 +1,9 @@
+# Scala 2.13.0-M5
+ProblemFilters.exclude[MissingTypesProblem]("akka.http.caching.scaladsl.CachingSettings$")
+
+# CachingSettings public/impl classes split
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.http.caching.scaladsl.CachingSettings.fromSubConfig")
```

Comment:
Hmm, that's because that method was publicly implemented in `CachingSettings`. Strictly, it's a binary incompatibility but I guess we can risk it because it's unlikely to be used directly.